src/pages/resources/recipes/ipc.md

```diff
@@ -0,0 +1,130 @@
+# Inter Plugin Communication
+
+UXP allows communication between plugins that are installed in the application. This is particularly handy when you know a certain task is already automated by another plugin and you would like to invoke it instead of duplicating the effort. But make sure the user is not caught by surprise. Your plugin should call out such dependencies to ensure flawless functioning of your plugin.
+
+`Plugin Manager` module provides APIs that will help establish the connection. But before we take a look at an example, be sure to be well-versed in these topics
+- Plugin entrypoints //TODO Add a link
+- Manifest permission module //TODO Add a link
+
+Additionally, you will need
+- certain basic knowledge of the plugin you are going to communicate with. This includes:
+    - the plugin `id`
+    - available `entrypoints`
+    - the structure of arguments to be passed (if applicable)
+- `enablePluginCommunication` permission for your plugin
+
+<InlineAlert variant="info" slots="text"/>
+
+Evidently, the concepts of inter-plugin communication is not applicable to **UXP scripts**.
+
+## Example 
+
+**Caller Plugin**
+```json
+/* manifest.json */ 
+{
+    "requiredPermissions": {
+        "ipc": {
+            "enablePluginCommunication": true
+        }
+    }
+}
+```
+```js
+function communicateWithAnotherPlugin() {
+    try {
+        const allPlugins = pluginManager.plugins;
+        const plugin = Array.from(allPlugins).find(plugin => plugin.id === "com.adobe.example.coolPlugin");
+        if (plugin && plugin.enabled) {
+            console.log('All commands:', plugin.manifest.commands);
+            console.log('All panels:', plugin.manifest.panels);
+    
+            /* Show the plugin panel; Note that panels can only be made visible -- you can't ask to hide the panel */
+            plugin.showPanel("simplePanel");
+    
+            plugin.invokeCommand("simpleCommand");
+            
+            /* Send an argument to the command  */
+            const name = {
+                firstName: "John",
+                lastName: "Doe"
+            };
+            plugin.invokeCommand("commandWithInput", name);
+        } else {
+            // prompt the user to install or enable the plugin before trying again
+        }
+    } catch (e) {
+        console.error(e);
+    }
+} 
+```
+//TODO: What does 'plugin.enabled' mean? When can the value be false?
```

Comment:
@kareemmyk

Reply:
```plugin.enabled``` is managed by option enable/disable available in plugins market place. 
Below is screenshot for reference


<img width="722" alt="Screenshot 2023-06-14 at 9 36 05 AM" src="https://github.com/AdobeDocs/uxp-indesign/assets/34368858/ce507a4a-38ef-4e10-9a3f-af4fd841b3f1">